server/risk_scores/risk_assessment.py

```diff
@@ -199,23 +187,25 @@ def get_current_risk_score(form: submit_schema.Form):
     return normalize_current_risk_score(risk_score)
 
 
-def get_risk_assessment(form: submit_schema.Form, timeframe) -> RiskAssessment:
-    total_risk_score = get_current_risk_score(form) + get_previous_risk_score(form, timeframe)
+def get_risk_assessment(form: submit_schema.Form, timeframe: int) -> RiskAssessment:
+    score_from_current_incident = get_current_risk_score(form)
+    score_from_prev_incidents = get_previous_incidents_risk_score(
+        form, timeframe)
+    risk_assessment = risk_score_combiner.combine_risk_scores(
+        score_from_current_incident, score_from_prev_incidents)
+
     form_dict = form.dict()
 
-    for i, (max_percent, assessment) in enumerate(assessment_ranges):
-        if total_risk_score <= max_percent:
-            if i >= minimum_email_score_index and credentials.PYTHON_ENV != "development":
-                email_dict = {
-                    "staff_name": form_dict["staff_name"],
-                    "client_primary": form_dict["client_primary"],
-                    "risk_score": assessment.value,
-                    "risk_score_raw": total_risk_score
-                }
-                email_high_risk_alert(email_dict)
-            return assessment
-    else:
-        return RiskAssessment.UNDEFINED
+    if risk_assessment >= minimum_email_score_index and credentials.PYTHON_ENV != "development":
+        email_dict = {
+            "staff_name": form_dict["staff_name"],
+            "client_primary": form_dict["client_primary"],
+            "risk_score": assessment.value,
```

Comment:
Since `assessment.value` is no longer defined, you can just change this to `risk_assessment`.
Also now that the combined risk score is no longer a numeric value, you might want to get the corresponding string (LOW, MEDIUM, or HIGH) from `RiskAssessment` and pass it into the email